modules/local/cellrangerarc/count/main.nf

```diff
@@ -0,0 +1,73 @@
+process CELLRANGERARC_COUNT {
```

Comment:
Any chance we can get this into nf-core/modules? What's the state of https://github.com/nf-core/modules/pull/3576?

Reply:
Basically it was approve by @maxulysse for merging. The modules are failing because of internal docker issues.